libraries/bsg_manycore_config.h

```diff
@@ -58,8 +58,8 @@ extern "C" {
         #define HB_MC_CONFIG_MAX_BITWIDTH_ADDR 30
         #define HB_MC_CONFIG_MAX_BITWIDTH_DATA 32
 
-        #define HB_MC_CONFIG_VCORE_BASE_X 0
-        #define HB_MC_CONFIG_VCORE_BASE_Y 2
+        #define HB_MC_CONFIG_VCORE_BASE_X 16
+        #define HB_MC_CONFIG_VCORE_BASE_Y 8
```

Comment:
is this read from the rom now?

Reply:
I guess it can be. I kicked this can down the road because having multiple manycore pods will change what this even means.

Reply:
OK, push to #667